ui/tests/acceptance/sync/secrets/overview-test.js

```diff
@@ -53,16 +52,21 @@ module('Acceptance | sync | overview', function (hooks) {
         },
       };
     });
+
     this.server.post('/sys/activation-flags/secrets-sync/activate', () => {
       return {};
     });
-    await click(ts.navLink('Secrets Sync'));
-    assert.dom(ts.overview.optInBanner).exists('Opt-in banner is shown');
+
+    await visit('/vault/sync/secrets/overview');
+
+    assert.dom(ts.overview.optInBanner).exists();
     await click(ts.overview.optInBannerEnable);
-    assert.dom(ts.overview.optInModal).exists('Opt-in modal is shown');
-    assert.dom(ts.overview.optInConfirm).isDisabled('Confirm button is disabled when checkbox is unchecked');
+
+    assert.dom(ts.overview.optInModal).exists('modal to opt-in and activate feature is shown');
     await click(ts.overview.optInCheck);
     await click(ts.overview.optInConfirm);
+
+    assert.dom(ts.overview.optInModal).doesNotExist('modal is gone once activation has been submitted');
```

Comment:
Maybe we can add a bit more to ensure the flow continues to work? Something like click "Create destination" then you're on the destination create page.

Reply:
agreed! just added coverage for clicking on create & verifying the URL 😀